packages/react-ui-components/src/components/SchemaForm/Widgets/CheckboxesWidget/utils.ts

```diff
@@ -90,6 +94,10 @@ export const toggleSelectedOptions = <T extends CheckboxOption>(
 
 			return allOptions.map(({ value }) => value);
 		}
+
+		if (selectedOptions.includes(selectedOptionValue) && selectedOptions.length === minItems) {
```

Comment:
Can you explain what the purpose of this is change? It might not be necessary 🤔 
From my understanding, before de-selecting an option, we must ensure the selected options' minimum constraint is met. Due to this, if the selected options count is already at the minimum, we must not perform any action and leave it as it is.
This is what is being done between #L104 and #L108.